style.css

```diff
@@ -1006,3 +1093,1378 @@
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin-top: 148px;
+    display: flex;
+    margin-left: 16px;
+  }
+
+  .page7 .description2 {
+    display: none;
+  }
+
+  .page7 .button1 {
+    display: none;
+  }
+
+  .page7 .description5-desktop {
+    width: 337px;
+    height: 72px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    display: flex;
+    margin-left: 16px;
+    margin-top: 24px;
+  }
+
+  .page8 {
+    background-image: url(images/Img_Placeholder_3_desktop.png);
+    height: 435px;
+    width: 369px;
+  }
+
+  .page8 .heading6-desktop {
+    width: 337px;
+    height: 38px;
+    font-family: var(--font3);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin-left: 16px;
+    margin-top: 190px;
+    display: flex;
+  }
+
+  .page8 .heading2 {
+    display: none;
+  }
+
+  .page8 .button1 {
+    display: none;
+  }
+
+  .page8 .description6-desktop {
+    width: 337px;
+    height: 72px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    display: flex;
+    margin-top: 24px;
+    margin-left: 16px;
+  }
+
+  .lang ul {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 12px;
+    gap: 12px;
+    list-style: none;
+    margin-left: 16px;
+    width: 236px;
+    height: 40px;
+  }
+
+  .lang .html a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    width: 56px;
+    height: 40px;
+    background: rgba(255, 255, 255, 0.24);
+    text-decoration: none;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .lang .bootstrap a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    width: 95px;
+    height: 40px;
+    background: rgba(255, 255, 255, 0.24);
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .lang .Ruby a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 10px 12px;
+    width: 61px;
+    height: 40px;
+    background: rgba(255, 255, 255, 0.24);
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .about {
+    width: 1440px;
+    height: 1195px;
+    padding: 120px 24px 0 24px;
+    margin-top: 120px;
+    background-color: #1c1a19;
+    background-position: bottom 87% right;
+    background-image: url(images/illustration_about_me_desktop.png);
+    background-repeat: no-repeat;
+  }
+
```

Comment:
- [ ] The background graphics in the about section are not appearing also. Please check again the image you are referencing if it exists.
![image](https://user-images.githubusercontent.com/90222110/188199880-dc8ffcd8-6e8a-4d57-9554-6b7f9514a88b.png)